### DISCUSSION TITLE
Jest testing AE v3 alpha has been released, with Qv2 support

### DISCUSSION THREAD

**Discussion by IlCallo** [quasarframework/quasar#10174] [Announcements]
Hi everyone!

A new major version for Jest testing AE has been released as alpha

See release notes here: https://github.com/quasarframework/quasar-testing/releases/tag/unit-jest%40v3.0.0-alpha.3

**Comment by Jajiso** [comment]
Hello

I'm having some problems when using this version and I don't know if something is missing.

The problem is with this test:
<img width="563" alt="1" src="https://user-images.githubusercontent.com/39906822/128149647-fa655b4f-8fbf-4806-9008-404bd4654a38.PNG">

I'm trying to test if the data chages its value when clicking the button but is giving me this error:
<img width="715" alt="2" src="https://user-images.githubusercontent.com/39906822/128149865-9b9fbe43-9516-410e-af1d-53af5d092111.PNG">

And this is the component related to the test:
<img width="163" alt="3" src="https://user-images.githubusercontent.com/39906822/128149978-033913c3-8c3b-42d5-8ea5-0c0a734e7dac.PNG">
<img width="192" alt="4" src="https://user-images.githubusercontent.com/39906822/128149996-13d31c5d-5ada-4a26-b3b1-848dadc66f0f.PNG">
<img width="204" alt="5" src="https://user-images.githubusercontent.com/39906822/128150005-05e07346-b394-4a72-9cd7-045c1d65c923.PNG">

Someone can help me with this please?

Thank you!

**Reply by IlCallo** [reply]
Can you create a repro and open an issue on https://github.com/quasarframework/quasar-testing repo?
